eslint-rules/lib/rules/no-direct-import.js

```diff
@@ -85,8 +87,45 @@ module.exports = {
       }
     }
 
+    function reportDirectRequire(node, rule) {
```

Comment:
Couldn't you use the utils as we've discussed? Did it not work?

Reply:
I think the utils we saw do a little more than I need. They aggregate the components and from everywhere and all I need to do is just prohibit the usage of a library.

Reply:
It does not sound like a good enough reason to duplicate the code.
Using the util should remove a lot of code.

Reply:
I could use the utils some how but the code will be much uglier. It aggregates all the imports and returns an object. Checking the imports after that is kind of ugly because I will need to go over all the keys of the object and check if its in the rules. I guess I could do that but its not clear. I guess thats why it wasn't used in the import declaration before. I'm also not sure it will exactly do what I need and will not capture other cases as well.